packages/core/test/scxml.test.ts

```diff
@@ -254,16 +245,16 @@ const testGroups = {
     // 'test354.txml', // conversion of namelist not implemented yet
     'test355.txml',
     'test364.txml',
-    // 'test372.txml', // microstep not implemented correctly
+    // 'test372.txml', // microstep not implemented correctly for final states
     'test375.txml',
     // 'test376.txml', // executable blocks not implemented
     'test377.txml',
     // 'test378.txml', // executable blocks not implemented
     'test387.txml',
-    // 'test388.txml', // computed historyValue not being available immediately after exiting states for the following synchronous enterStates
+    'test388.txml',
```

Comment:
hm, when this was fixed? 🤔 just wondering if maybe this doesn't pass false positively now?

Reply:
Probably with the microstep refactor; if it is indeed a false positive, we'll see it fail with further refactoring, or see similar tests fail. Not sure if it's worth it to deeply inspect a passing test.